src/main/java/org/dbflute/intro/app/logic/core/FlutyFileLogic.java

```diff
@@ -44,4 +47,24 @@ public void writeFile(File textFile, String content) {
             throw new UncheckedIOException("Cannot write the text file: " + textFile, e);
         }
     }
+
+    /**
+     * Open directory by filer. (e.g. finder if mac, explorer if windows)
+     * Use OS command.
+     *
+     * @param dir open directory name (NotEmpty)
+     * @throws DirNotFoundException 
```

Comment:
@jflute 
How should javadoc here be written?
I forgot.；；

Reply:
e.g.

o first argument is not "name" so fix it like this:
```
@param dir File object for opened directory. (NotEmpty)
```

o write DirNotFoundException description
```
@throws DirNotFoundException When ...
```

Reply:
fixed.

74a062b743fcaa698c62d205e1057b47d2584756